libs/content/workspace-ffi/SwiftWorkspace/Sources/Editor/iOSMTK.swift

```diff
@@ -853,15 +851,17 @@ public class iOSMTK: MTKView, MTKViewDelegate {
         redrawTask?.cancel()
         self.isPaused = output.redraw_in > 100
         if self.isPaused {
-            let redrawIn = Int(truncatingIfNeeded: output.redraw_in)
+            var redrawIn = Int(truncatingIfNeeded: output.redraw_in)
             
-            if redrawIn != -1 {
-                let newRedrawTask = DispatchWorkItem {
-                    self.setNeedsDisplay(self.frame)
-                }
-                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(redrawIn), execute: newRedrawTask)
-                redrawTask = newRedrawTask
+            if redrawIn == -1 {
```

Comment:
Could this be less weird easily

Reply:
Made it less weird by using a `UInt64`, so I don't need to truncate.